react/README.md

```diff
@@ -1,36 +1,50 @@
-# Boxwise react client 
+# Readme
 
-This front-end project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This frontend project of Boxtribute was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## All available Create-React-Scripts
+## Table of Contents
 
-In the project directory, you can run:
+1. [Contribution Guidelines](../CONTRIBUTING.md)
+2. [Development Set-up](#development-set-up)
+   1. [Set-up pre-commit](#set-up-pre-commit)
+   2. [Install node and yarn](#install-node-and-yarn)
+   3. [Linting and Formatting in VSCode](#linting-and-formatting-in-vscode)
+3. [Note about yarn and Docker](#note-about-yarn-and-docker)
+4. [Testing][#testing]
```

Comment:
second [ ] should be ( ) for correct markdown

Reply:
done